index.js

```diff
@@ -115,8 +115,9 @@ module.exports = bundler => {
             const copyTo = dir.staticOutDir
                 ? path.join(bundleDir, dir.staticOutDir)
                 : bundleDir;
-
-            copyDir(path.join(pkg.pkgdir, dir.staticPath), copyTo);
+            // merge global exclude glob with static path exclude glob
+            const excludeGlob = (config.excludeGlob || []).concat((dir.excludeGlob || [])); 
```

Comment:
Make sure there's always an array, and we're not concating undefined

Reply:
ok